src/PhpSpec/CodeAnalysis/AccessInspectorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpspec\CodeAnalysis;
+
+interface AccessInspectorInterface
```

Comment:
I did consciously decide to follow the existing naming convention, however if you want to start moving towards not using `*Interface` for a v3 release, I'm happy to start now.